examples/space/auto_weighting.py

```diff
@@ -0,0 +1,57 @@
+"""Example demonstrating the usage of the ``auto_weighting`` decorator."""
```

Comment:
This reads more like a test than an example, is it possible to make it easier on the eyes?